dit/dit_exporter.py

```diff
@@ -123,6 +125,10 @@ def task(group, group_id, subgroup, subgroup_id, task, task_id, data):
     if not data:
         return
 
+    if id_only:
```

Comment:
Move the declarations

```
created_at = data.get('created_at')
...
logbook = data.get('logbook', [])
```

after line 153 and group this conditional with the next one

```
# write
if id_only:
    _write('%s/%s/%s' % (group_id, subgroup_id, task_id))
    return
if _options.get('compact_header'):
    _write(_ca('[%s/%s/%s]' % (group_id, subgroup_id, task_id)) + ' ' +
           _cc(names_to_string(group, subgroup, task)))
else:
    global _group_string, _subgroup_string
    if _group_string:
        _write(_group_string)
        _group_string = None
    if _subgroup_string:
        _write(_subgroup_string)
        _subgroup_string = None
    _write(_cc('[%s/%s/%s] %s' % (group_id, subgroup_id, task_id, task)))
```